lua/telescope/actions/init.lua

```diff
@@ -272,6 +272,23 @@ function actions.close_pum(_)
   end
 end
 
+--- Copy current entry command to a specific register or unnamed plus as default
+--- Don't let user use  blackhole register (_), let them choose instead
+---@param prompt_bufnr number: The prompt bufnr
+---@param opts table: options to pass to the action
+---@field reg: the register which the content will be sent
+function actions.copy_command_to_reg(prompt_bufnr, opts)
+  opts = opts or {} -- ensure opts is a table if nil
+  opts.reg = utils.get_default(opts.reg, "+")
+  if opts.reg == "_" then
+    opts.reg = vim.fn.input "Register > "
```

Comment:
Also my bad, this should either validate that the user only has passed one char (get first char then or something) or use `vim.fn.getchar` instead.